### PR TITLE
[ko] Update incorrect expression dev-1.23-ko.2

### PR DESCRIPTION
**Pull request by yyyoungha**
For maxSurge, round up is correct, but for maxUnavailable, I think round down is correct.

see below for more details:

1) func ResolveFenceposts
https://github.com/kubernetes/kubernetes/blob/914475e2e9f483ca5c96e50ce664343a37721d0e/pkg/controller/deployment/util/deployment_util.go#L849-L868

2) func GetScaledValueFromIntOrPercent
https://github.com/kubernetes/apimachinery/blob/276a8a7530a30f42c82b1f0fa2261a905130bf97/pkg/util/intstr/intstr.go#L146-L162

### CODE REVIEW

**Comment by ianychoi** [comment]
It is duplicated with https://github.com/kubernetes/website/pull/32297 but @yyyoungha Would you help reflect a recently merged patch https://github.com/kubernetes/website/pull/32297 to this working working branch? That contribution would be highly appreciated

File: content/ko/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ API 버전 `apps/v1` 에서는 `.spec.selector` 와 `.metadata.labels` 이 설
 
 `.spec.strategy.rollingUpdate.maxUnavailable` 은 업데이트 프로세스 중에 사용할 수 없는 최대 파드의 수를 지정하는 선택적 필드이다.
 이 값은 절대 숫자(예: 5) 또는 의도한 파드 비율(예: 10%)이 될 수 있다.
-절대 값은 반올림해서 백분율로 계산한다.
+절대 값은 내림해서 백분율로 계산한다.
```
Comment (jihoon-seo) [inline content/ko/docs/concepts/workloads/controllers/deployment.md:1134]: @yyyoungha 기여 감사합니다 😊
https://github.com/kubernetes/website/pull/32297/files 를 보시면
L1147 에

```diff
-절대 값은 반올림해서 백분율로 계산한다. 기본 값은 25% 이다.
+절대 값은 올림해서 백분율로 계산한다. 기본 값은 25% 이다.
```

변경사항도 있는데,
이 PR에도 반영해 주실 수 있을까요? 😊

/remove-language de en es id it ja pt ru uk zh

**Comment by yyyoungha** [comment]
@ianychoi Sorry I accidentally merged the main branch and a lot of irrelevant labels were attached... Can I merge the main branch here (dev-1.23-ko.2](https://github.com/kubernetes/website/tree/dev-1.23-ko.2))? I tried to cherry-pick, but it seems the commit is in front.

File: content/ko/docs/concepts/workloads/controllers/deployment.md (hunk 1 above)
Comment (jihoon-seo) [reply content/ko/docs/concepts/workloads/controllers/deployment.md:1134]: /remove-area blog release-eng
/remove-sig release

Comment (yyyoungha) [reply content/ko/docs/concepts/workloads/controllers/deployment.md:1134]: Sure 😊

**Comment by jihoon-seo** [comment]
@yyyoungha 불필요한 레이블은 모두 지웠고, rebase 잘 해 주셔서 현재 PR에는 필요한 커밋만 있는 것 같습니다.
#32297 이 머지 완료된 점을 고려하여, LGTM 남깁니다 😊
/lgtm

**Comment by seokho-son** [comment]
@yyyoungha Thank you for the contribution !

/lgtm

Could you squash the commits into one commit ?
then, I will approve this PR.

**Comment by yyyoungha** [comment]
@seokho-son Got it. No problem.

**Comment by jihoon-seo** [comment]
감사합니다!
/lgtm
/approve

**Pull request merged**